tikv/region.go

```diff
@@ -246,8 +246,10 @@ type RegionOptions struct {
 }
 
 type RegionManager interface {
-	GetRegionFromCtx(ctx *kvrpcpb.Context) (*regionCtx, *errorpb.Error)
+	GetRegionFromCtx(ctx *kvrpcpb.Context) (*regionCtx, *errorpb.Error, string, uint64)
```

Comment:
Can we use a separate method to get the store address and id?

Reply:
OK, updated.